docs/pages/styled-jsx/+Page.mdx

```diff
@@ -1,13 +1,20 @@
 import { Example, UiFrameworkExtension, CommunityNote } from '../../components'
 import { Link } from '@brillout/docpress'
 
-<CommunityNote tool="styled-jsx" url="https://github.com/vercel/styled-jsx" hasExtension={false} />
+<CommunityNote tool="styled-jsx" url="https://github.com/vercel/styled-jsx" hasExtension="react" />
 
-Example:
+## Manual integration
+
+If you use <Link href="/vike-react">`vike-react`</Link>, you can manually integrate styled-jsx by using:
+- <Link href="/onBeforeRenderHtml" doNotInferSectionTitle />
+- <Link href="/Wrapper" doNotInferSectionTitle />
+- <Link href="/onAfterRenderHtml" doNotInferSectionTitle />
+
+Example of using Vike with styled-jsx:
  - <Example timestamp="2023.07" repo="jeremypress/vite-ssr-styled-jsx" />
```

Comment:
Do the example use `vike-react` or only Vike core?

Reply:
That one is still using `vite-plugin-ssr`.

I forgot to add my example—I'll add it now